### ISSUE TITLE
[SCREENREADER]: Focus is not being returned to the Edit mailing address button on Cancel from confirmation modal

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6470]
## Description

**❗️ Must**

Focus is not being returned to the `Edit mailing address` button when I enter an overridable address a second time and then press `Cancel` from the second confirmation modal. This could cause screen reader users to be confused if their changed address was entered correctly or not. Screenshots attached below, outlining how this can be re-created.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/5865

## Point of Contact

**VFS Point of Contact:** _Jennifer, Trevor can consult too_

## Acceptance Criteria

- [ ] As a screen reader user, I want to hear the `Edit mailing address, button` read aloud when I press the Cancel button from the second validation modal

## Environment

* Windows 10
* IE11
* JAWS
* [Test Validation Case 3](https://dsvavsp.testrail.io/index.php?/cases/view/170)
* [Test validation case 5](https://dsvavsp.testrail.io/index.php?/cases/view/175)

## Steps to Recreate

1. Log into test user 56's `/profile`
2. Click the `Edit` button for mailing address
3. Enter a nonsense address, or a valid, overridable address
4. Press the `Update` button
5. Press the `Edit it` or `Edit address` button from the validation modal
6. Enter a further nonsense address, or update your valid overridable address
7. Press the `Update` button again
8. Press the `Cancel` button from the second validation modal
9. Verify the focus is not on the `Edit` button for mailing address (no yellow halo)

## Possible Fixes (optional)
Use the React `focus` helper to set focus properly on the end of this use flow.

## WCAG or Vendor Guidance (optional)
* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2020-03-04 at 9 45 58 AM](https://user-images.githubusercontent.com/934879/75903859-d801cc80-5e07-11ea-96ca-ebf7258f86ba.png)

---
![Screen Shot 2020-03-04 at 9 46 10 AM](https://user-images.githubusercontent.com/934879/75903873-ddf7ad80-5e07-11ea-9bd6-f2a3d2bf5fac.png)

---
![Screen Shot 2020-03-04 at 9 46 18 AM](https://user-images.githubusercontent.com/934879/75903897-e2bc6180-5e07-11ea-8d36-2d99de61a551.png)

---
![Screen Shot 2020-03-04 at 9 46 27 AM](https://user-images.githubusercontent.com/934879/75903910-e7811580-5e07-11ea-9792-8c9e8478560d.png)

**Comment by Samara-Strauss** [comment]
@bradl3yC @mshea0606 let's get this prioritized this sprint *if* we can. But let's focus on some of the other launch-required to-dos before coming back to this. We can discuss more in grooming.

**Comment by bradl3yC** [comment]
@jenstrickland this should be ready to validate as soon as its in staging ~20-30 minutes

**Comment by jenstrickland** [comment]
@bradl3yC I'll set a reminder and check at 3:30pm ET.

**Comment by jenstrickland** [comment]
Validated this, the focus returns to the Edit button after multiple edits and cancel.
